package.json

```diff
@@ -125,5 +134,33 @@
   "engine": {
     "node": "~10.14.2",
     "npm": "~6.4.1"
+  },
+  "detox": {
+    "test-runner": "jest",
+    "configurations": {
+      "ios.release": {
+        "binaryPath": "./ios/build/Build/Products/Release-iphonesimulator/emurgo.app",
+        "build": "xcodebuild -workspace ios/emurgo.xcworkspace -configuration release -scheme emurgo -sdk iphonesimulator -derivedDataPath ios/build",
+        "type": "ios.simulator",
+        "name": "iPhone 11"
+      },
+      "android.emu.mainnet.debug": {
```

Comment:
I don't think the debug Android build is used anywhere. Is there a benefit to keeping this as an option for the e2e test? For example maybe it's easier to debug an E2E test issue with this build?

Reply:
I don't think we should use it for CI but it can be practical in a situation in which somebody wants to run the test but doesn't have the signing keys needed for a release build. That's why I just added the setup config but didn't bother to add a `yarn` shortcut.